### DISCUSSION TITLE
In SO v2.4.10 .... Where to configure the Suricata HOME_NET permanently ? Thanks

### DISCUSSION THREAD

**Discussion by plaotec** [Security-Onion-Solutions/securityonion#11204] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

Installed SO v2.4.10 .... But not getting any Suricata Alerts... SO-STATUS all ok .... It is using the default HOME_NET.......Where to configure the Suricata HOME_NET permanently and  the syntax ? Thanks

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by plaotec** [comment]
setting up the home_net is not part of the S.O v2.4.10 setup dialog

**Comment by plaotec** [comment]
tried editing the /opt/so/conf/suricata/suricata.yaml file, to change the homenet, but after change, and restarted so-suricata, the homenet went back to the default values.

**Comment by plaotec** [comment]
found it in the GUI now Administration –> Configuration –> suricata –> config
–> vars –> address-groups –> HOME_NET.

After changing home_net, still no alerts , rebooted SO VM, still no alerts

**Comment by reyesj2** [comment]
Have you tried the steps outlined in the docs https://docs.securityonion.net/en/2.4/suricata.html#troubleshooting-alerts

Can you verify you're ingesting data that you expect to trigger an alert? 

Are you seeing any zeek / suricata metadata collected? Just no alert triggered?

**Comment by plaotec** [comment, marked as answer]
after i found where to change the home net in the GUI..I saw the suricata alerts...i am running in ESXi v7.0

but after our lab automation system saved the VM and relaunch the VM, I don't see suricata alerts anymore and the home net is still correct

another strange thing is like kibana connections logs, if i set time range to 12 hrs , i see the connections logs number increasing....if i set time range like to last 5 minutes, last 1 hr, I see 0 connections logs 

thanks
